data/transport/actions/transport.yaml

```
@@ -499,3 +540,20 @@ transport . âge nouvelle voiture électrique . occasion:
 
   par défaut: oui
 # L'action disparait quand l'impact climat devient positif
+
+transport . voiture . trajets longs . arrêter:
+  titre: Privilégier le train aux longs trajets en voiture
+  icônes: 🚆🚗
+  formule: (empreinte . usage * trajets longs . part / voyageurs) - (voiture . km * trajets longs . part * train . impact par km * voyageurs)
```

```suggestion
  formule: ( (empreinte . usage * trajets longs . part) / voyageurs) - (voiture . km * trajets longs . part * train . impact par km * voyageurs)
```
Explicite la priorité dans le calcul